src/components/events/create/CreateEventContainer.js

```diff
@@ -2,28 +2,12 @@
 /* eslint-disable react/no-unused-state */
 import React, {Component} from 'react';
 import  PropTypes from 'prop-types';
-import  Validator from 'validator';
 import {connect} from 'react-redux';
 import moment from 'moment';
-import CreateEventForm from './EventForm';
-import {create} from '../../redux/actions/events';
-import {addFlashMessage} from '../../redux/actions/flashMessages';
-
-const validate=(data) => {
-    const errors = {};
-    if(Validator.isEmpty(data.name)) errors.name = 'name can\'t be blank';
-    if(Validator.isEmpty(data.description)) errors.description = 'description can\'t be blank';
-    if(Validator.isEmpty(data.category)) errors.category = 'category can\'t be blank';
-    if(Validator.isEmpty(data.location)) errors.location = 'location can\'t be blank';
-    return errors;
-};
-const formatDate = eventDate => {
-    const date = new Date(eventDate);
-    const month = date.getMonth() + 1;
-    const day = date.getDate();
-    const year = date.getFullYear();
-    return [year, month, day].join('-');
-};
+import CreateEventForm from '../EventForm';
+import {create} from '../../../redux/actions/events';
```

Comment:
There should be a space after the first brace and before the last, for  purposes